components/Appointment.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
```

Comment:
Please see imports order, absolute paths, and line spaces:


"use client";
// line space

import Image from "next/image";
import { Box, Container, Paper, Stack, Typography } from "@mui/material";
import { dosis } from "@/fonts/Dosis";
import appointment from "@/public/images/appointment.png"; // alphabetical fonts before public
import Agenda from "./Agenda"; // => @/components/Agenda

// line space
const Appointment = () => {